### ISSUE TITLE
NVDA is full of errors in Chromium-based browsers. It does not read under the mouse cursor.

### ISSUE THREAD

**Issue by wise1453** [nvaccess/nvda#11045]
I am using the NVDA beta version. I am a low vision user. I write the problems respectively.

It does not read buttons on all sites. Go to any site. Place the mouse cursor over the buttons. You will notice that most buttons are not read by NVDA.

It does not read texts next to the selection boxes. It is necessary to click on the selection box. Reads after clicking. It does not focus on search boxes. Although the site is loaded, it does not focus. It is necessary to press the E key many times. It does not focus when you click with the mouse. Sometimes it doesn't focus on writing areas. It is necessary to click on the text field first. Then it is necessary to press the TAB key once. It again focuses on pressing SHIFT + TAB.

Based on Firefox Chromium. The text under the mouse cursor does not read paragraph by paragraph. Reads line by line. It becomes an accented text in a paragraph or sentence. Problems occur when there is accented text. Reads to accented text. When it hovers over highlighted text, it reads it. It does not read the text behind it. It is necessary to move the mouse cursor over the text behind it. He does not read articles on some sites. Problems occur most in Edge browser. Chromium based: Does not read some text in Chrome, Edge, Brave, Opera and Yandex browsers. It does not sound most buttons. It does not speak the selection boxes. It does not sound Radio Buttons. It does not sound unless you open pop-up menus and selected menu. It does not also speak the tag. Many errors occur when using Edge. Right click with the mouse cursor. Do it anywhere on the page. In the menu that opens, move the mouse cursor. Sometimes NVDA shuts down. NVDA crashes. It is about right click menu in Chrome, Edge, Opera, brave, yandex browsers. Right click. Open the menu. Hover the mouse over it. Descriptions of the menus are read. While reading, it does not stop after every menu description. It accumulates in a row. He continues to read. You close the menu. NVDA still continues to voice the menu content from where it left off. The above problem also applies to upper menus and upper sections. NVDA works with delay. You pass the relevant object. NVDA lags behind. Reads 10 objects ahead. This is an example. 10-6-7-5 is not important before the object. The important thing is that NVDA does it all the time. Delayed reads. It accumulates and reads on top of each other. There are more problems. I report all of these issues over time. NVDA 2012 version was very good. Those posted after that release are bad. Mouse support is reduced. Mouse support is ending. There is regression. Please pay attention to my calls and notifications.

**Comment by Brian1Gaff** [comment]
I think it would be useful if you refer to the issue numbers you raised on 
this. I guess what is needed is a partially sighted developer to run through 
the list you mention and look for some commonality.
 I'm sure somebody will be along in a moment to ask you too fill in one of 
the templates so a at least folk can start from the situation you have.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users

**Comment by Adriani90** [comment]
@wise1453 your description is not very easy to read. Please in the future, stick to the template and describe one by one how your issue is reproducible. In software development it is not very easy to investigate if you have much plain text containing almost no details.
Don't get me wrong, thank you very much for reporting. I just suggest to make your description short and concentrate on the point, giving a structured overview. This is why the template has been created.

Comming back to your issue, please search on NVDA's repository under issues for the word mouse, then you will find already many issues which have been created already. Most of them incluse also what you already describe.
It is not just an issue cause by NVDA, since NVDA 2012 also Chrome, Firefox and other browsers changed alot. Thus, it's not only related to NVDA.

Finally, one suggestion for you: please go to NVDA menu under tools and run the COM registration fixing tool. Is it better after running that tool?

**Comment by Adriani90** [comment]
And please post the version of NVDA you are testing with as well as the version of all other programs you see this issue in. Including also the version of your Windows. Thanks.

**Comment by Adriani90** [comment]
See for example #10650 or #9515.

**Comment by Adriani90** [comment]
Closing in favor of the issues referenced above and other issues related to this.
Moreover, I checked with last alpha on Edge Chromium 81 after #11050 has been merged and at least moving the mouse around does not cause errors.
Regarding NVDA freezin in context menus, there is already issue #10895.
